planet_sheet/config.tex

```diff
@@ -15,37 +15,37 @@
     ItalicFont = *-Italic
 ]{CourierPrime}
 % supplemental information (lighter-colored text)
-\definecolor{supplemental}{HTML}{C0C0C0}
```

Comment:
This color was too light and for some B&W printer it was barely visible so I propose a darken color.